robot/impl/local_robot_test.go

```diff
@@ -3482,3 +3482,21 @@ func TestCustomResourceBuildsOnModuleAddition(t *testing.T) {
 	)
 	rtestutils.VerifySameResourceNames(t, r.ResourceNames(), expectedResources)
 }
+
+func TestSendTriggerConfig(t *testing.T) {
+	// Tests that the triggerConfig channel buffers exactly 1 message and that
+	// sendTriggerConfig does not block whether or not the channel has
+	// capacity.
+	ctx := context.Background()
+	logger := logging.NewTestLogger(t)
+
+	r := setupLocalRobot(t, ctx, &config.Config{}, logger)
+	r.Close(ctx)
```

Comment:
Why are we `Close`ing immediately in this test? Won't that cause `sendTriggerConfig` to be a noop (surprised this test passes)?

Reply:
Oops @maximpertsov asked before me 😆 .

Reply:
answered here https://github.com/viamrobotics/rdk/pull/3968#discussion_r1643314785